apache-maven/src/assembly/maven/conf/settings.xml

```diff
@@ -244,14 +244,43 @@ under the License.
       </properties>
     </profile>
     -->
+    <profile>
+      <id>maven:core:central-repo</id>
+      <repositories>
+        <repository>
+          <id>central</id>
+          <name>Central Repository</name>
+          <url>https://repo.maven.apache.org/maven2</url>
+          <snapshots>
+            <enabled>false</enabled>
+          </snapshots>
+        </repository>
+      </repositories>
+
+      <pluginRepositories>
+        <pluginRepository>
+          <id>central</id>
+          <name>Central Repository</name>
+          <url>https://repo.maven.apache.org/maven2</url>
+          <snapshots>
+            <enabled>false</enabled>
+          </snapshots>
+          <releases>
+            <updatePolicy>never</updatePolicy>
+          </releases>
+      </pluginRepository>
+    </pluginRepositories>
+    </profile>
   </profiles>
 
   <!-- activeProfiles
    | List of profiles that are active for all builds.
    |
+  -->
   <activeProfiles>
-    <activeProfile>alwaysActiveProfile</activeProfile>
-    <activeProfile>anotherAlwaysActiveProfile</activeProfile>
+    <activeProfile>maven:core:central-repo</activeProfile>
```

Comment:
I don't like this approach. Let's invest time to do it correct, i.e. without profiles

Reply:
I can't offer anything better until settings don't change. Take your time and let me know.